frappe/desk/doctype/route/route.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
There are a few problems with this.
1. Route is a very generic name, we can call it "Desk Route"
1. Having a separate doctype just to store an extra 1:1 map of route is wasteful, Instead a checkbox to toggle to route in the shortcut child doc is better.